.gitconfig.local

```diff
@@ -0,0 +1,5 @@
+[user]
```

Comment:
it seems this file was added to repository mistakenly. Please remove it

Reply:
Yes it was an error, i deleted this file, thanks